### ISSUE TITLE
Palo Alto Networks Input quoted string with comma’s

### ISSUE THREAD

**Issue by F47991** [Graylog2/graylog-plugin-integrations#21]
## Description
The Palo Alto Input doesn’t ignore commas in a quoted string, and therefore doesn’t index the fields properly.

## Steps To Reproduce

1. Create a Palo Alto Networks Input
2. Forward Logs from a Palo Alto System via TCP to Graylog
3. Logs that include commas within a quoted string shifts the fields in the index

Example message:
```
1,2019/04/25 15:32:04,009900009999,SYSTEM,globalprotect,0,2019/04/25 15:32:04,,globalprotectportal-auth-succ,GP_PortalAdm_Optional,0,0,general,informational,"GlobalProtect portal user authentication succeeded. Login from: 10.0.0.1, Source region: DK, User name: pre-logon, Auth type: client certificate.",16657523,0x0,0,0,0,0,,PA-VM
```
## Environment

- Graylog Version: 3.0.1
- Elasticsearch Version: 6.7.1
- MongoDB Version: 4.0.9

**Comment by florianvolle** [comment]
Closing this as this has been fixed in #16 and will be in the upcoming 3.0.2 release.

**Comment by danotorrey** [comment]
Hi @F47991,
Thanks for the report of the issue. As @florianvolle mentioned, this will be fixed in the 3.0.2 release, which should occur soon (most likely within a week or two).

I have uploaded a fixed Graylog Plugin Integrations 3.0.1 release jar that you can use until the 3.0.2 release is available. Please note that this link will only be valid through the end of the week.
https://drive.google.com/file/d/13_aOb8ZriZjgTpooOV9JJVpetGrGVKPI/view?usp=sharing
